ftplugin/bzl.vim

```diff
@@ -53,13 +53,16 @@ let &l:tabstop = s:save_tabstop
 
 setlocal formatoptions-=t
 
+let b:undo_ftplugin .= " | setlocal fo<"
```

Comment:
Isn't `b:undo_ftplugin` not necessarily defined yet, in which case this would fail?

Reply:
It's defined in the sourced Python ftplugin sourced at line 43.

Reply:
Oh, right! Could you put a comment above this line to note it's initially defined in ftplugin/python.vim?

Reply:
Done